### PR TITLE
Proof of concept for WISH #3786

### PR DESCRIPTION
**Pull request by gltewalt**

### CODE REVIEW

File: red.r [hunk 1]
```diff
@@ -525,11 +526,13 @@ redc: context [
 		]
 		exe: safe-to-local-file exe
 
-		either gui? [
-			gui-sys-call exe any [all [file form-args file] ""]
-		][
-			if with [repend exe [" " form-args file]]
-			sys-call exe								;-- replace the buggy CALL native
+    unless quiet? [
```
Comment (greggirwin) [inline red.r]: Needs leading tabs instead of spaces.

File: red.r [hunk 2]
```diff
@@ -436,8 +436,9 @@ redc: context [
 	]
 
 	run-console: func [
-		gui?	[logic!]
-		debug?	[logic!]
+		gui?	   [logic!]
+		debug?	 [logic!]
+		nol?     [logic!] 								;-- no launch
```
Comment (greggirwin) [inline red.r]: `nol?` is...not a great name.

**Comment by greggirwin** [comment]
Seems like it would be better to make the option an affirmative, rather than a negative. i.e. `launch-console`.

File: red.r (hunk 1 above)
Comment (gltewalt) [reply red.r]: Hmm, thought it was tabs...

File: red.r (hunk 2 above)
Comment (gltewalt) [reply red.r]: Yeah, I changed to `quiet?` not too long after that version.

**Comment by gltewalt** [comment]
> 
> 
> Seems like it would be better to make the option an affirmative, rather than a negative. i.e. `launch-console`.

Do you mean change the default behavior for the consoles to compiling only? Launch only if a flag is supplied?

**Comment by greggirwin** [comment]
No, I mean the default is still to launch the console, but *internally* that default is true. If a "--no-console" flag is given, `launch-console` gets set to false.

**Comment by gltewalt** [comment]
I had to re fork my red repo, so I can't make changes to the wish PR anymore.

For comment, these are the changes in line with your suggestion:
https://github.com/gltewalt/red/commit/ba2c63c2accaf8bbca1fb6d2af841f4fef5b4af0

**Comment by gltewalt** [comment]
Will resubmit at a later time

**Pull request closed without merging**